oregon-a/basket-dev/worker-deploy.yaml

```
@@ -48,366 +37,226 @@ spec:
         - name: SECURE_SSL_REDIRECT
-          valueFrom:
-            secretKeyRef:
-              key: secure-ssl-redirect
-              name: basket-dev-v107-env
+          value: "False"
         - name: SENTRY_DSN
           valueFrom:
             secretKeyRef:
               key: sentry-dsn
-              name: basket-dev-v107-env
+              name: basket-dev-secrets
         - name: SFDC_PASSWORD
           valueFrom:
             secretKeyRef:
               key: sfdc-password
-              name: basket-dev-v107-env
+              name: basket-dev-secrets
         - name: SFDC_SEC_TOKEN
           valueFrom:
             secretKeyRef:
               key: sfdc-sec-token
-              name: basket-dev-v107-env
+              name: basket-dev-secrets
         - name: SFDC_USERNAME
           valueFrom:
             secretKeyRef:
               key: sfdc-username
-              name: basket-dev-v107-env
+              name: basket-dev-secrets
         - name: STATSD_CLIENT
-          valueFrom:
-            secretKeyRef:
-              key: statsd-client
-              name: basket-dev-v107-env
+          value: django_statsd.clients.normal
         - name: STATSD_HOST
-          valueFrom:
-            secretKeyRef:
-              key: statsd-host
-              name: basket-dev-v107-env
+          value: datadog-statsd.datadog.svc.cluster.local
         - name: STATSD_PREFIX
-          valueFrom:
-            secretKeyRef:
-              key: statsd-prefix
-              name: basket-dev-v107-env
+          value: basket-dev-oregon-b
         - name: SUBHUB_OPP_RECORD_TYPE
-          valueFrom:
-            secretKeyRef:
-              key: subhub-opp-record-type
-              name: basket-dev-v107-env
+          value: 012R0000000dQ4dIAE
         - name: SUPERTOKEN
           valueFrom:
             secretKeyRef:
               key: supertoken
-              name: basket-dev-v107-env
+              name: basket-dev-secrets
         - name: SYNC_KEY
           valueFrom:
             secretKeyRef:
               key: sync-key
-              name: basket-dev-v107-env
+              name: basket-dev-secrets
         - name: TASK_LOCKING_ENABLE
-          valueFrom:
-            secretKeyRef:
-              key: task-locking-enable
-              name: basket-dev-v107-env
-        - name: WORKFLOW_RELEASE
-          valueFrom:
-            secretKeyRef:
-              key: workflow-release
-              name: basket-dev-v107-env
-        - name: WORKFLOW_RELEASE_CREATED_AT
-          valueFrom:
-            secretKeyRef:
-              key: workflow-release-created-at
-              name: basket-dev-v107-env
-        - name: WORKFLOW_RELEASE_SUMMARY
-          valueFrom:
-            secretKeyRef:
-              key: workflow-release-summary
-              name: basket-dev-v107-env
-        image: 127.0.0.1:5555/basket-dev:v107
+          value: "True"
+        - name: WSGI_NUM_WORKERS
+          value: 4
```

```suggestion
          value: "4"
```

To prevent a type error